predicators/envs/base_env.py

```diff
@@ -342,3 +342,7 @@ def get_event_to_action_fn(
         """
         raise NotImplementedError("This environment did not implement an "
                                   "interface for human demonstrations!")
+
+    def get_state(self) -> State:
+        """Get the current state of this environment."""
+        return self._current_state
```

Comment:
let's do `self._current_state.copy()` to avoid any changes to the environment's internal state